packages/integration-sdk-testing/src/jest.ts

```diff
@@ -183,14 +183,15 @@ export function toMatchGraphObjectSchema<T extends Entity>(
   { _class, schema }: ToMatchGraphObjectSchemaParams,
 ) {
   // Copy this so that we do not interfere with globals.
-  // NOTE: The data-model should actuall expose a function for generating
+  // NOTE: The data-model should actually expose a function for generating
   // a new object of the `IntegrationSchema`.
   const dataModelIntegrationSchema = dataModel.IntegrationSchema;
-  _class = Array.isArray(_class) ? _class : [_class];
+  const classArray = Array.isArray(_class) ? _class : [_class];
```

Comment:
I think there is other usage of `_class` that will no longer be normalized to an array (e.g. here: https://github.com/JupiterOne/sdk/blob/a9bf99cd267dd23cc4a06467913119248f0e3b41/packages/integration-sdk-testing/src/jest.ts#L221). Is that going to cause issues? I think that all of our tests currently assume an array because of that. Maybe this behavior is better, but might cause breaking behavior in our existing tests?

Reply:
Yes, it looks like it would. I baked these out of this commit so we just address #333.